experiments/configs/eval_configs/nlp_eval_config.yaml

```diff
@@ -0,0 +1,11 @@
+model: hf-causal
+model_args: "pretrained=/fsx/path/checkpoint" # update
+tasks: "lambada_standard"
```

Comment:
Is `lambada_standard` the only nlp task available in this fork? do we have other tasks that we can run like arc / hellaswag?

Reply:
I didn't put them in so we were not duplicating the same eval between the sections, and arc/hellaswag are both used in the `experiments/scripts/run_n_shot_benchmarks_eval.py` script. Do you think its worth running them here with 0 shot for extra reference even if performance might be low with no icl? 
I wasn't really sure what to use besides lambada that would be most informative for us. One choice could be to run the whole (or part of) the `experiments/scripts/run_n_shot_benchmarks_eval.py` script (maybe just excluding the slowest tasks?)

Reply:
oh no that's fine. I just hadn't understood the _full evaluation_ is done with both the list of 0-shot configs and the n_shot script. I thought the n-shot script was separate. That makes a lot of sense 👍.

Reply:
Yes I'm not sure if we should be running the whole thing for the run_n_shot_benchmarks_eval.py, we might need to check how long it takes now that we have increased the batch size. It needs multiple machines to run and since we will be using most for the training it will effectively not be parallelised over machines if it we run it during training. We could maybe comment out hendrycks full (and maybe run a subset 0 shot like in the stem script) and also hellaswag just to run at the end because the rest are faster.